extensions/cornerstone/src/init.js

```diff
@@ -267,10 +272,10 @@ export default async function init({
       contextMenuHandleClick
     );
 
-    eventTarget.addEventListener(EVENTS.STACK_VIEWPORT_NEW_STACK, evt => {
-      const { element } = evt.detail;
-      utilities.stackPrefetch.enable(element);
-    });
+    eventTarget.addEventListener(
```

Comment:
This is a small performance enhancement that prevents continually adding new callbacks - there is exactly one, and it knows how to call the nested enable events.  It is only related to the rest of the changes in that it made debugging a LOT easier not to see multiple calls being made.